.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change
+
+_Select the type of your PR_
+
+- [ ] Bugfix
+- [ ] Enhancement / new feature
+
+
+### Description
+
+_Please describe your pull request_
+
+### Checklist
+
+_Please go through this checklist and make sure all aplicable tasks have been done_
+
+- [ ] Write tests!
+- [ ] Make sure all tests pass!
+- [ ] Check coding style!
```

Comment:
Since we don't have a documented coding standard and we're not (yet) enforcing anything in CI, I think we should drop this.

Reply:
I/m planning with your PR for checkstyle which seems like it might be done soon ;-)